src/scripts/pagerduty.coffee

```diff
@@ -808,6 +808,8 @@ module.exports = (robot) ->
             ]
           return cb(users, schedule)
         cb(json.entries[0].user, schedule)
+      else
+        cb(null, schedule)
```

Comment:
@kuba-kubula I was only able to find this pattern in `pager me schedule` and this one is working well (it only shows the entries with your user assigned). Any other place I might have omitted?